src/evolutionaryAlgorithm/EvolutionSimulator.py

```diff
@@ -11,8 +11,8 @@
 
 
 class EvolutionSimulator:
-    initializator: InitializatorInterface
-    recombinator: RecombinatorInterface
+    initializator = InitializatorInterface
+    recombinator = RecombinatorInterface
```

Comment:
WTF?

Reply:
nie, nie to jest tak jak ty zrobiłaś a w jednym momencie się "bawiłe" bo nie kompilowało się a szukałem problemów więc nevermind